src/Akeneo/Category/back/Application/Query/GetCategoryChildrenIdsInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Category\Application\Query;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface GetCategoryChildrenIdsInterface
```

Comment:
In DDD context, adding Interface in the name of the interface does not seem right as we are adding a technical term in the application layer
```suggestion
interface GetCategoryChildrenIds
```